### PR TITLE
Contextually create quip.

### PR DESCRIPTION
**Pull request by hashamali**
## Overview

When viewing a circle feed, if the user selects compose quip, the circle is preselected when they reach the circle selection fragment.

### CODE REVIEW

File: app/src/main/java/it/quip/android/activity/QuipitHomeActivity.java [hunk 1]
```diff
@@ -220,13 +220,18 @@ private void displayDefaultQuipStream() {
...
 
             @Override
             public void onCreateCircle() {
                 createCircle();
             }
+
+            @Override
+            public void onCreateQuipInCircle(Circle circle) {
+                // TODO: them thangs
+            }
```
Comment (dbuscaglia) [inline app/src/main/java/it/quip/android/activity/QuipitHomeActivity.java]: Whats this thangs

File: app/src/main/java/it/quip/android/activity/CreateCircleActivity.java [hunk 2]
```diff
@@ -38,6 +38,12 @@ protected void onCreate(Bundle savedInstanceState) {
         }
     }
 
+    @Override
+    public void onBackPressed() {
+        finish();
+        overridePendingTransition(R.anim.zoom_in, R.anim.slide_down);
```
Comment (jcomo) [inline app/src/main/java/it/quip/android/activity/CreateCircleActivity.java]: When we create the circle, we should also make a transition back to the last activity. Like the other activities that now have `animateOut`, we should do it for this one too

File: app/src/main/java/it/quip/android/adapter/SearchArrayAdapter.java [hunk 3]
```diff
@@ -108,4 +110,8 @@ private void hideChecked(ImageView iv) {
         iv.setVisibility(View.INVISIBLE);
     }
 
+    private boolean isPreselected(T value) {
+        return (null != mPreselectedValues) && (mPreselectedValues.contains(value.getObjectId()));
```
Comment (jcomo) [inline app/src/main/java/it/quip/android/adapter/SearchArrayAdapter.java:114]: #thangs

**Comment by jcomo** [comment]
Looks good

**Pull request merged**